frontend/src/components/SearchBar.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useApolloClient } from '@apollo/client';
+import gql from 'graphql-tag';
```

Comment:
graphql-tag is not the correct library, should come form graphql. (Is this AI generated, no tutorials even refer to this library with regards to Apollo client)